spec/API_specification/data_type_functions.md

```diff
@@ -0,0 +1,63 @@
+# Data type functions
+
+> Array API specification for data type functions.
+
+A conforming implementation of the array API standard must provide and support the following data type functions.
+
+<!-- NOTE: please keep the constants in alphabetical order -->
+
+## Objects in API
+
+(finfo)=
+### finfo(type)
+
+Defines the machine limits for floating point types.
+
+#### Parameters
+
+-   **type**: _Union\[ float, &lt;dtype&gt, instance ]_
```

Comment:
I'm not sure about adding builtins (`int`, `float`) here. NumPy seems to translate that to `int64` / `float64`, but that's not quite correct. I'd just type it as `<dtype>` instead.

Reply:
Yes! I agree with this, I'll leave it just as _dtype_